demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -137,17 +131,23 @@ public void onReceived(List<DownloadBatchStatus> downloadBatchStatuses) {
         bindViews();
     }
 
-    VersionOneDatabaseCloner.CloneCallback cloneCallback = new VersionOneDatabaseCloner.CloneCallback() {
+    private final VersionOneDatabaseCloner.CloneCallback cloneCallback = new VersionOneDatabaseCloner.CloneCallback() {
         @Override
         public void onUpdate(String updateMessage) {
             databaseCloningUpdates.setText(updateMessage);
         }
     };
 
-    Migrator.Callback migrationCallback = new Migrator.Callback() {
+    private final MigrationCallback migrationCallback = new MigrationCallback() {
         @Override
-        public void onUpdate(MigrationStatus migrationStatus) {
-            databaseMigrationUpdates.setText(migrationStatus.status().toRawValue());
+        public void onUpdate(final MigrationStatus migrationStatus) {
+            Handler handler = new Handler(getMainLooper());
```

Comment:
Do we still need this? We wrap the callback in a main thread handler in `DownloadMigrator`

Reply:
Oh snap 😂

Reply:
You saw nothing 😂 b354991